src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -158,4 +159,40 @@ Scalar<DataVector> area_element(
     const tnsr::i<DataVector, 3, Frame>& normal_one_form,
     const DataVector& radius,
     const tnsr::i<DataVector, 3, Frame>& r_hat) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Spin function of a 2D `Strahlkorper`.
+ *
+ * \details See Eqs. (2) and (10)
+ * of https://arxiv.org/abs/1708.07325.
+ * Specifically, computes \f$\Omega = \epsilon^{AB}\nabla_A\omega_B\f$,
+ * where capital indices index the tangent bundle of the surface and
+ * where \f$\omega_\mu=(K_{\rho\nu}-K g_{\rho\nu})h_\mu^\rho s^\nu\f$ is
```

Comment:
This comment sounds like little omega is the curl of the the angular momentum density of the surface - from what I understand, isn't big Omega the curl and little omega the angular momentum density?

Reply:
Big Omega is one of the terms in the spin integral, which is the curl of little omega. I'll add a note explaining the spin function's physical significance: it's proportional to the horizon vorticity, and it's part of the integral for the spin. The other part of the spin integral is an approximate Killing vector.